src/main/java/cinema/model/Movie.java

```diff
@@ -0,0 +1,76 @@
+package cinema.model;
+
+import java.util.Objects;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Movie {
```

Comment:
in the scope of this task only Movie should be present

Reply:
Well, I'm not going to remove these entities anyway...